src/popup/i18n/locales/de.json5

```diff
@@ -0,0 +1,162 @@
+{
+  // Component App
+  app: {
+    // Error message used if an error is raised during initialization of background scripts. Prompt users to contact the feedback.
+    bgInitError: "Ein schwerwiegender Fehler ist aufgetreten: \"{msg}\". Falls dieser Fehler immer noch aufkommt nach einem Neustart ihres Browsers, kontaktieren Sie @:app.settings.contact.value",
+    // Page Settings
+    settings: {
+      // Prompt users to go to the previous page
+      back: "Zurück",
+      // Name of page where users can change settings
+      label: "Einstellungen",
+      // Prompt users to enable desktop notification
+      enableNtf: {
+        // Label for enableNtf
+        label: "Schalte Benachrichtigungen ein",
+        // Description for enableNtf
+        description: 'Benachrichtigt dich, falls ein (un)geplanter Livestream startet ("Livestream startet in 10 Minuten")'
+      },
+      // Prompt users to enable 30-hour clock
+      enable30Hours: {
+        // Label for enable30Hours
+        label: "Aktiviere 30 Stunden Uhr",
+        // Description for enable30Hours
+        description: "Der Tag startet um 06:00, erweitere 24:00 bis 29:59 um 0:00 AM - 5:59 AM des nächsten Tag zu repräsentieren"
+      },
+      // Prompt users to sync settings between different devices with the same browser account logged in
+      shouldSyncSettings: {
+        // Label for shouldSyncSettings
+        label: "Sync Einstellungen",
+        // Description for shouldSyncSettings
+        description: "Teile dieselben Einstellungen auf mehrere Geräte mit demselben Browser Konto"
+      },
+      // Prompt users to switch the UI language
+      language: {
+        // Label for language
+        label: "Language",
+        // Description for language
+        description: "Ändere die UI Sprache",
+        // Supported locales, where keys are type of i18n.LanguageCode and values are translated into the according locale
+        locales: {
+          en: "English",
+          ja: "日本語",
+          "zh-CN": "简体中文",
+		  de: "Deutsch"
```

Comment:
Extra `\t`s are inserted here. I think we should remove them ;-)

```suggestion
          de: "Deutsch"
```

Reply:
i dont see what you actually mean

![notepad++_EYZ9y3O5vE](https://user-images.githubusercontent.com/80579142/114901941-99658080-9e15-11eb-8a16-10c327aeb094.png)

Unless you want me to remove the indent from every key value pair by one. But the de is on the same line as all other

Reply:
I could also put the de above "zh-CN"

Reply:
I‘m sorry to be unclear here and let me clarify the reason why I'm strongly against the usage of `\t` or a 'tab' character:
1. Plain spaces are widely used in this project to manage indents
1. Width of a space equals width of a normal character, especially when we are displaying with a 'monospaced' font
1. Width of a tab depends on editors, some are '1 tab = 2 spaces', some are '1 tab = 4 spaces'(notepad++), 8 spaces(github webpage as seen),  even 16 spaces(windows notepad).

The rules above would result in that when tabs and spaces are used together, the indents and visual alignment are instable between different edit tools/different developers.
![0](https://user-images.githubusercontent.com/23452609/114905893-0d228000-9e4c-11eb-8ce5-710a0502bff5.png)

So for the consistancy, we encourage replacing are tabs with spaces.

Reply:
Tabs and spaces are both invisible but we can find their different width by selecting them with the cursor.

Reply:
Ok gotchu' will edit rn and push